### ISSUE TITLE
Check --notify-work values are URLs

### ISSUE THREAD

**Issue by joojis** [CodeChain-io/codechain#723]
Currently, the below command works without any error or warning.
```sh
./codechain --notify-work 127.0.0.1:3333 ...
```

**Comment by hea9549** [comment]
Should this work in the http protocol? Or is it a matter of not specifying the protocol?

**Comment by joojis** [comment]
@hea9549 Just http is ok to me. :)

**Comment by sgkim126** [comment]
@joojis What's about stratum?

**Comment by joojis** [comment]
@sgkim126 Is it different? I'm not sure. It seems that string is only used for HTTP request.

**Comment by sgkim126** [comment]
@joojis As far as I remember, stratum uses TCP socket directly.
But I'm not sure stratum support for notify-work is done.

@kseo Do you know the current status of it?

**Comment by redongjun** [comment]
@sgkim126 You are right, stratum is used TCP socket for notify-work and submitting a solution. 
so '--notify-work' arg isn't related to stratum.
